docs/src/SUMMARY.md

```diff
@@ -2,10 +2,12 @@
 
 [Introduction](./Introduction.md)
 
-- [Installation Instructions](./installation/README.md)
-  - [Installing on a Raspberry Pi](./installation/Raspberry-Pi.md)
+- [Installation & Build Instructions](./installation/README.md)
+  - [Installing on a Raspberry Pi 32-bit/ARM32](./installation/Raspberry-Pi-32_ARM32.md)
+  - [Installing on a Raspberry Pi 64-bit/ARM64](./installation/Raspberry-Pi-64_ARM64.md)
   - [Installing on Ubuntu (from source)](./installation/Ubuntu.md)
```

Comment:
Since you deleted that file, maybe remove that line too?

Reply:
Of course. :)